### DISCUSSION TITLE
Geography for elementary flows

### DISCUSSION THREAD

**Discussion by ErCollao** [USEPA/Federal-LCA-Commons-Elementary-Flow-List#188] [Ideas]
Hi all! We've been working on mapping some impact assessment methods to FEDEFL, and we're noticing one feature becoming more and more prominent: geographical identification of elementary flows (particularly land use). I was wondering if there's any plan on including it in FEDEFL?

**Comment by bl-young** [comment]
We do not anticipate that individual elementary flows would have locations as part of our schema. However, we anticipate that factors for LCIA methods might have specific locations and so that is an option in the schema for LCIAformatter (see [here](https://github.com/USEPA/LCIAformatter/blob/master/format%20specs/LCIAmethod.md)). We are currently working on a geographically explicit method for freshwater eutrophication which will be the first opportunity to test that implementation.

Do you see that as a viable alternative for your needs?

**Comment by cmutel** [comment]
Please please please do not go down this path - it is the wrong way to get regionalized LCA scores.

1. It is conceptually wrong: Elementary flows represent an abstract idea. They do *not* have a spatiotemporal context - this is inherited from the process which consumes the resource or releases the emission. 
2. It is methodologically wrong. Regionalized LCIA maps have a *different spatial scale* than inventory maps (and each implementation can use a different map from previous implementations of the same impact category). We also have [elegant math](https://eartharxiv.org/repository/view/5270/) to allow for this change of spatial scale from impact assessment to inventory. There is no way to add locations to elementary flows in any reasonable fashion while preserving the resolution of the impact assessment maps.
3. It is a data nightmare. There is no fixed spatial scale for inventory or impact assessment spatial units, and impact assessment maps vary by impact category. Just for watersheds you would need to add around 1e4 copies of each water-related flow.
